src/styles/blocks/catalog.scss

```
@@ -12,14 +12,21 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    margin-bottom: $standard-margin;
+
+    transition: transform 300ms, color 300ms;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

don't remember create variables for colors
```suggestion
      color: $c-someColor;
```